packages/core/src/features/st-scope.ts

```diff
@@ -21,19 +20,38 @@ export const hooks = createFeature<{ IMMUTABLE_SELECTOR: ImmutablePseudoClass }>
         if (!isStScopeStatement(atRule)) {
             return;
         }
-        if (!atRule.params) {
-            context.diagnostics.report(diagnostics.MISSING_SCOPING_PARAM(), { node: atRule });
+        const selectorAst = parseSelectorWithCache(atRule.params);
+        const isValidSelector =
```

Comment:
isValidSelector? I'm not sure what do you test here

Reply:
that the params, when parsed as a selector list:
- have at least 1 selector
- each selector doesn't have an invalid node
- each selector have some actual selector (not just comments)

Reply:
removed the check and allowed any params to be analyzed as a selector.